bindata/network/kuryr/004-daemon.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kuryr-cni
+  namespace: openshift-kuryr
+  annotations:
+    kubernetes.io/description: |
+      This DaemonSet launches the kuryr-daemon component.
+spec:
+  template:
+    metadata:
+      name: kuryr-cni
+      labels:
+        app: kuryr
+        component: network
+        type: infra
+        openshift.io/component: network
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      hostNetwork: true
+      serviceAccountName: kuryr
+      priorityClassName: system-node-critical
+      initContainers:
+      - name: install-cni-plugins
+        image: {{ .NodeImage }}
```

Comment:
This may be OK for now but kuryr should be using its own image here, which contains whatever CNI plugins it needs, rather than assuming that the openshift-sdn image will have the right ones.

Reply:
@danwinship: Okay, please help me on this one. So this is taken directly from openshift-ansible where playbooks of SDN's were responsible for injecting all the "general" plugins, in particular loopback, which was required for CNI to work. Is that not the case anymore?

Reply:
Some day, the SDN image will go away entirely (when we've fully migrated to ovn-kubernetes). So in the long run you should not depend on it.

At the moment you could also get the `loopback` and `host-local` plugins from the `container-networking-plugins-unsupported` image, although that seems like a bug; they should presumably be in the `container-networking-plugins-supported` image instead...

Reply:
Fair enough, so I guess I can leave it like it is for now and update once this is sorted out.